taskcluster/scriptworker-task.yml

```diff
@@ -0,0 +1,30 @@
+build:
+  template_file: simple-task.tyml
+  dependencies:
+    - "test-python_27-linux-amd64-opt"
+    - "test-python_34-linux-amd64-opt"
+    - "test-python_35-linux-amd64-opt"
+    - "test-python_36-linux-amd64-opt"
+    - "test-nodejs_4x-linux-amd64-opt"
+    - "test-nodejs_5x-linux-amd64-opt"
+    - "test-nodejs_6x-linux-amd64-opt"
+  allowed:
+    # FIXME: Switch to "push" and "refs/tags/" when ready.
+    # - "push"
+    - "pull_request.opened"
+    - "pull_request.synchronize"
+    - "pull_request.reopened"
+  # FIXME: Switch to "push" and "refs/tags/" when ready.
+  # ref_match: "refs/tags/"
+  ref_match: "refs/heads/"
+  routes:
+    - "notify.irc-channel.${notifications.irc}.on-exception"
+    - "notify.irc-channel.${notifications.irc}.on-failed"
+  artifacts_deps:
+    - "darwin-amd64-cpu-opt"
+    - "linux-amd64-cpu-opt"
+    - "linux-amd64-gpu-opt"
```

Comment:
We have to remove the CUDA builds from the dependency list here, as the package is too big to be uploaded for now. When https://github.com/pypa/packaging-problems/issues/86#issuecomment-345223919 is fixed we can put it back again.